x-pack/plugins/ml/server/routes/indices.ts

```diff
@@ -18,15 +18,14 @@ export function indicesRoutes({ router, mlLicense }: RouteInitialization) {
    * @api {post} /api/ml/indices/field_caps
    * @apiName FieldCaps
    * @apiDescription Retrieves the capabilities of fields among multiple indices.
+   *
+   * @apiSchema (body) indicesSchema
```

Comment:
These params look ok to me, but I don't see the `apiName` in the generated markdown:

![image](https://user-images.githubusercontent.com/7405507/79235443-85361e80-7e63-11ea-80f6-7e1b8307b72a.png)

Reply:
Fixes, it was the case because of missing annotation in `@api`